interfaces/IBF-dashboard/src/app/auth/auth.service.ts

```diff
@@ -112,7 +114,12 @@ export class AuthService implements OnDestroy {
     this.router.navigate(['/']);
   };
 
-  private onLoginError = (error) => {
+  private onLoginError = async ({ error }) => {
+    const toast = await this.toastController.create({
+      message: `Authentication Failed: ${error?.message}`,
+      duration: 2000,
```

Comment:
I think this can be a bit longer, especially because the toast is not very visually popping out, so the user might need a second to locate it. I would put it at 5 seconds maybe?